pennylane/devices/tests/test_measurements.py

```diff
@@ -211,10 +264,10 @@ def test_paulix_pauliy(self, device, tol, skip_if):
         def circuit():
             qml.RX(theta, wires=[0])
             qml.RX(phi, wires=[1])
-            qml.RX(phi, wires=[2])
+            qml.RX(varphi, wires=[2])
             qml.CNOT(wires=[0, 1])
             qml.CNOT(wires=[1, 2])
-            return qml.expval(qml.PauliX(wires=0)), qml.expval(qml.PauliX(wires=2))
+            return qml.expval(qml.PauliX(wires=0) @ qml.PauliY(wires=2))
```

Comment:
Bug uncovered...

Reply:
How come this test was passing before? 😨

Reply:
I think there was never a device natively supporting tensor observables that we ran the test suite with.

Reply:
Wait, wouldn't this run on `default.qubit`?

Reply:
Interesting question - but default.qubit did not previously have `tensor_observables` set to True!

Reply:
It inherited it from ``QubitDevice``, I believe:

```python
    @classmethod
    def capabilities(cls):
        capabilities = cls._capabilities
        capabilities.update(model="qubit", tensor_observables=True)
        return capabilities
```

So it should have been running?

Reply:
I changed the default value for ``tensor_observables`` in ``QubitDevice``, it was ``False`` :)  But the logic is implemented in the device, so I think this is better.